### ISSUE TITLE
Sorting Algorithms

### ISSUE THREAD

**Issue by ayan-biswas0412** [ayan-biswas0412/AlgoDocs#4]
Here is a possible list of sorting algorithms that need to be implemented 
please refrain from adding algorithms in the same language, check the corresponding issues before working

- [ ] Selection Sort #49 
- [ ] Bubble Sort 
- [ ] Recursive Bubble Sort
- [ ] Insertion Sort  #46 
- [ ] Recursive Insertion Sort
- [ ] Merge Sort #47 
- [ ] Iterative Merge Sort
- [ ] Quick Sort
- [ ] Iterative Quick Sort
- [ ] Bitonic Sort
- [ ] Stooge Sort
- [ ] Gnome Sort
- [ ] Add your own (comment below please)

Please comment below first if you are interested to implement any of the above algorithms, then please open a new issue using the issue template and then open a PR corresponding to your issue

**Comment by ishan-gupt** [comment]
i can post selection sort,bubble sort,insertion sort

**Comment by ayan-biswas0412** [comment]
> i can post selection sort,bubble sort,insertion sort

please open separate issues first, then start working on them, I would suggest taking one by one 
Open 1 issue -> submit PR -> then open another

**Comment by JASKAMAL22** [comment]
Hey @ayan-biswas0412, I would like to work on bubble sort in python or recursive bubble sort.

**Comment by TishaJhabak1014** [comment]
@ayan-biswas0412 hey, plz let me contribute with merge sort in python.
Thank you

**Comment by ayan-biswas0412** [comment]
> @ayan-biswas0412 hey, plz let me contribute with merge sort in python. Thank you

Yeah sure go ahead, your contribution will help this project to grow more.

**Comment by TishaJhabak1014** [comment]
> > @ayan-biswas0412 hey, plz let me contribute with merge sort in python. Thank you
> 
> Yeah sure go ahead, your contribution will help this project to grow more.

Shall I create an issue for that?

**Comment by ayan-biswas0412** [comment]
> > > @ayan-biswas0412 hey, plz let me contribute with merge sort in python. Thank you
> > 
> > 
> > Yeah sure go ahead, your contribution will help this project to grow more.
> 
> Shall I create an issue for that?

sure

**Comment by Tanisha1721** [comment]
I would like to do bubble sort and selection sort in java. May I please work on this?
